calico/getting-started/kubernetes/windows-calico/kubernetes/standard.mdx

```diff
@@ -283,19 +283,18 @@ cd {{rootDirWindows}}
 
 :::note
 
-The installer initializes the Windows vSwitch, which can cause a short connectivity outage as the networking stack is reconfigured. After running that command, you may need to:
+ The installer initializes the Windows vSwitch, which can cause a short connectivity outage as the networking stack is reconfigured. After running that command, you may need to:
 
-- Reconnect to your remote desktop session.
-- Restart `kubelet` and `kube-proxy` if they were already running.
-- If you haven't started `kubelet` and `kube-proxy` already, you should do so now. The quickstart scripts provided in the {{prodname}} package provide an easy way to do this. {{prodname}} requires `kubelet` to be running to complete its per-node configuration (since Kubelet creates the Kubernetes Node resource).
+ - Reconnect to your remote desktop session.
+ - Restart `kubelet` and `kube-proxy` if they were already running.
+ - If you haven't started `kubelet` and `kube-proxy` already, you should do so now. The quickstart scripts provided in the {{prodname}} package provide an easy way to do this. {{prodname}} requires `kubelet` to be running to complete its per-node configuration (since Kubelet creates the Kubernetes Node resource).
 
-  :::
-
-  :::note
+:::
 
-  After you run the installer, do not move the directory because the service registration refers to the path of the directory.
+:::note
 
-  :::
+ After you run the installer, do not move the directory because the service registration refers to the path of the directory.
+:::
```

Comment:
Missing space before `:::`

Reply:
@ctauchen Sorry, I did not understand which space you are referring to.

<img width="994" alt="image" src="https://user-images.githubusercontent.com/37767083/213860824-84fe6c1f-dcd8-480d-adea-6a51b6979738.png">

Reply:
There should be a blank line after 296. Between the note text and the closing `:::`